infrastructure/prod/anonymize_sql.py

```diff
@@ -0,0 +1,48 @@
+import os
+
+os.system('pg_dump prod > prod.sql')
+
+prod = open('prod.sql', 'r')
+anonomized_prod = open('anonomized_prod.sql', 'w')
+
+anonymizing_user = False
+anonymizing_phone_numbers = False
+
+lines = prod.readlines()
+
+for line in lines:
+  newLine = line
+
+
```

Comment:
Is it just the device I am looking at this on, or are these if statements not indented?